charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller-deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
       labels:
         app: csi
         role: controller
+        garden.sapcloud.io/role: controlplane
```

Comment:
What do you think about using 
```yaml
              matchExpressions:
              - key: networking.gardener.cloud/to-shoot-apiserver
                operator: In
                values:
                - allowed
```

instead of
https://github.com/gardener/gardener/blob/3cbde2d2dc25fbad2916682966b18f988fe57b77/charts/seed-bootstrap/charts/dependency-watchdog/templates/endpoint-configmap.yaml#L16-L20 ?

At least for some time, till all know extensions migrate from `garden.sapcloud.io/role` to `gardener.cloud/role`.

The reason is that components that must talk with the shoot's API already have this label and it can be used by the DWD to easily find them.